src/Display/DisplayConsts.h

```diff
@@ -0,0 +1,4 @@
+#pragma once
+
+// 'klic_logo', 128x64px
+extern const unsigned char bitmap_klic_logo[];
```

Comment:
Small suggestion: couldn't we add `stdint.h` header and use `uint8_t` instead?
```suggestion

#include <stdint.h>

// 'klic_logo', 128x64px
extern const uint8_t bitmap_klic_logo[];
```